osu.Game/OsuGame.cs

```diff
@@ -1197,6 +1199,15 @@ protected override void UpdateAfterChildren()
 
         private void screenChanged(IScreen current, IScreen newScreen)
         {
+            SentrySdk.ConfigureScope(scope =>
+            {
+                scope.Contexts[@"screen stack"] = new
+                {
+                    Current = newScreen.GetType().Name,
+                    Previous = current.GetType().Name,
```

Comment:
Should check for null I think? For entering into the first screen and exiting from the last screen.